travharv/web_discovery.py

```diff
@@ -73,11 +74,10 @@ def get_description_into_graph(
     :param format: indicating what format should be retrieved json-ld, turtle
     :type form: str
     :returns: the graph whith added discovered triples
-    :rtype: rdflib.Graph
+    :rtype: rdflib.Graph, dict
     """
 
-    if graph is None:
-        graph = Graph()  # create a fresh graph if you don't have it yet
+    graph = Graph()  # create a fresh graph if you don't have it yet
```

Comment:
comment outdated

but also unclear why you drop the opportunity we had before to pass and grow the graph - only initialise if none was there

I find this approach a regression to what we have